src/Microsoft.Diagnostics.TestHelpers/TestHost/TestDump.cs

```diff
@@ -26,7 +26,7 @@ public TestDump(TestConfiguration config)
             _serviceProvider.AddService<ISymbolService>(_symbolService);
 
             // Automatically enable symbol server support
-            _symbolService.AddSymbolServer(msdl: true, symweb: false, symbolServerPath: null, authToken: null, timeoutInMinutes: 0);
+            _symbolService.AddSymbolServer(msdl: true, symweb: false, timeoutInMinutes: 6, retryCount: 5);
```

Comment:
Is a 6 minute timeout a lot?

Reply:
I'm not sure. It is 2 minutes longer than the 4 minute default. It may still need to be adjusted along with the retry count in the future.